edx/analytics/tasks/load_internal_reporting_user_course.py

```
@@ -208,3 +212,47 @@ def run(self):
             raise
         finally:
             connection.close()
+
+
+class LoadUserCourseSummary(WarehouseMixin, VerticaCopyTask):
+    """
+
+    """
+    date = luigi.DateParameter()
+
+    @property
+    def partition(self):
+        """The table is partitioned by date."""
+        return HivePartition('dt', self.date.isoformat())  # pylint: disable=no-member
+
+    @property
+    def insert_source_task(self):
+        return ExternalURL(url=self.hive_partition_path('course_enrollment_summary', self.date))
```

Yeah, I considered borrowing that logic from user activity, however, I'm concerned about it silently using very stale data without us knowing about it.

We are missing a "strong warning" where it raises an alert that we actually see but proceeds with the load of the stale data.